### ISSUE TITLE
ESP8266_RELAY_BOARD project stacked on restarting

### ISSUE THREAD

**Issue by majercakdavid** [CHERTS/esp8266-devkit#13]
I wanted to try an ESP8266_RELAY_BOARD example but after flashing it and trying to boot it the device just stacks on restarting. Do you have any solution ?

**Comment by Def152** [comment]
+1

I'm using CHERTS devkit with sdk v1.3.0. Everything is compiling and flashing fine. But every 3 seconds esp is resetting with boot message (3, 7). I assumed it's power supply fault. I made another power supply 1A, 3,3V but this didn't helped. I also tried esp arduino and it's working like charm. So I thought it might be makefile. I'm using esp-12e with 32Mb, this is my makefile http://pastebin.com/p28REHFC for blink example
